src/components/annotations/draw.js

```diff
@@ -194,7 +194,7 @@ function drawRaw(gd, options, index, subplotId, xa, ya) {
 
     var isSizeConstrained = options.width || options.height;
 
-    var annTextClip = fullLayout._defs.select('.clips')
+    var annTextClip = fullLayout._topclips
```

Comment:
@alexcjohnson IMPORTANT: `Annotation` used to put its clips in `fulLayout._defs`, even though `Annotation` draws in `toppaper`. Clip ids are global to the page, so things still work fine (at in Chrome and FF). But, who knows maybe some other browsers don't apply clip ids outside the `<svg>` where the `<defs>` is located.

Reply:
Nice find! Definitely not the intent, thanks for cleaning that up!